### PR TITLE
update docs for python client

### PR DESCRIPTION
**Pull request by mangalaman93**

### CODE REVIEW

**Review by rarvikar — CHANGES_REQUESTED** [review]
Thanks for updating the docs !

Under the table for "Supported versions", shouldn't the pydgraph version be 21.3.x rather than 21.3.0 ? For example, a Dgraph v21.03.2 backend can work with multiple pydgraph versions v21.3.0, v21.3.1 and v21.3.2. So the table seems to indicate that v21 backends only work with a specific version of pydgraph - v21.3.0, which is misleading.  
Also, can we change Dgraph version in the table, from `21.X.Y` to `21.03.X` and `22.X.Y` to `22.0.X` ?

File: content/dql/clients/python.md [hunk 1]
```diff
@@ -34,75 +50,77 @@ servers in the same cluster allows for better distribution of workload.
...
+```
 
-If you want to connect to Dgraph running on your [Dgraph Cloud](https://cloud.dgraph.io) instance, then all you need is the URL of your Dgraph Cloud endpoint and the API key. You can get a client using them as follows:
+### Connecting To Dgraph Cloud
 
-```python
-import pydgraph
+If you want to connect to Dgraph running on [Dgraph Cloud](https://cloud.dgraph.io) instance, then
+all you need is the URL of your Dgraph Cloud endpoint and the API key. You can get a client using
+them as follows:
```
Comment (rderbier) [inline content/dql/clients/python.md:83]:
```suggestion
Create the 'client_stub' using the gRPC endpoint and the API key:
```

**Review by rderbier — APPROVED** [review]

**Comment by mangalaman93** [comment]
> For example, a Dgraph v21.03.2 backend can work with multiple pydgraph versions v21.3.0, v21.3.1 and v21.3.2.

We don't test for it, not sure whether we should say something like this in the README. Maybe it is easy enough to add tests against different Dgraph versions and build this table. Do you think that'd be useful?

**Comment by Rajakavitha1** [comment]
Hi @mangalaman93 !!! As a part of the docs revamp efforts we have removed this content from Docs and linked it to the repo to avoid duplication of update and maintenance efforts. https://github.com/dgraph-io/dgraph-docs/pull/610

**Pull request closed without merging**